exchanges/okx/okx.go

```diff
@@ -1077,79 +895,68 @@ func (ok *Okx) CreateRfq(ctx context.Context, arg CreateRfqInput) (*RfqResponse,
 		return nil, errInvalidCounterParties
 	}
 	if len(arg.Legs) == 0 {
-		return nil, errInvalidLegs
-	}
-	var resp []RfqResponse
-	err := ok.SendHTTPRequest(ctx, exchange.RestSpot, createRfqEPL, http.MethodPost, rfqCreateRfq, &arg, &resp, true)
-	if err != nil {
-		return nil, err
-	}
-	if len(resp) == 1 {
-		return &resp[0], nil
+		return nil, errMissingLegs
 	}
-	return nil, errNoValidResponseFromServer
+	var resp *RfqResponse
+	return resp, ok.SendHTTPRequest(ctx, exchange.RestSpot, createRfqEPL, http.MethodPost, "rfq/create-rfq", &arg, &resp, request.AuthenticatedRequest)
 }
 
 // CancelRfq Cancel an existing active Rfq that you has previously created.
-func (ok *Okx) CancelRfq(ctx context.Context, arg CancelRfqRequestParam) (*CancelRfqResponse, error) {
-	if arg.RfqID == "" && arg.ClientRfqID == "" {
-		return nil, errMissingRfqIDAndClientRfqID
+func (ok *Okx) CancelRfq(ctx context.Context, rfqID, clientRfqID string) (*CancelRfqResponse, error) {
+	if rfqID == "" && clientRfqID == "" {
+		return nil, errMissingOrderIDAndClientSuppliedID
 	}
-	var resp []CancelRfqResponse
-	err := ok.SendHTTPRequest(ctx, exchange.RestSpot, cancelRfqEPL, http.MethodPost, rfqCancelRfq, &arg, &resp, true)
-	if err != nil {
-		return nil, err
-	}
-	if len(resp) == 1 {
-		return &resp[0], nil
-	}
-	return nil, errNoValidResponseFromServer
+	var resp *CancelRfqResponse
+	return resp, ok.SendHTTPRequest(ctx, exchange.RestSpot, cancelRfqEPL, http.MethodPost, "rfq/cancel-rfq", &CancelRfqRequestParam{
+		RfqID:       rfqID,
+		ClientRfqID: clientRfqID,
+	}, &resp, request.AuthenticatedRequest)
 }
 
 // CancelMultipleRfqs cancel multiple active Rfqs in a single batch. Maximum 100 Rfq orders can be canceled at a time.
-func (ok *Okx) CancelMultipleRfqs(ctx context.Context, arg CancelRfqRequestsParam) ([]CancelRfqResponse, error) {
+func (ok *Okx) CancelMultipleRfqs(ctx context.Context, arg *CancelRfqRequestsParam) ([]CancelRfqResponse, error) {
+	if arg == nil {
```

Comment:
Acronym creation: MTC (Missing test coverage 😆 ) and the below len clientRfgIDs > 100 below